### DISCUSSION TITLE
Diversifier les systèmes liés aux pets

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1680] [Nouvelles idées]
Hello! Avec un petit groupe de gens on s'est accordé à dire que la vente de familier commence à être délaissé, plus personne achète par nécessité (sauf pour une mission de campagne) sûrement parceque les actifs commence à tous réclamer des 5 étoiles puisque les 4 étoiles sont partout.
Personnellement, ce qui m'inquiète c'est le côté guilde
La vente d'un pet 4 étoiles était un bon moyen de rapporter de l'xp de guilde en quantité mais désormais ce sont plus que des ventes par générosité qui sont effectués avec des personnes blindés souhaitant aider une guilde ce qui rend le level up de guilde plus ardu pour les commu hors du serv officiel. 

Nolitank et Muzan ont proposé de mettre en place un mini event au cours duquel on rencontre un marchant qui nous achète un pet de notre shelter de notre choix contre de l'xp de guilde (la valeur pourrait varier comme pour l'argent du roi).
Ça permettrais de ne pas avoir à dépendre des plus riches pour level up ta guilde de manière efficace.

Pagos m'a également confié l'idée de fight entre pet qui pourrais donner de l'xp de guilde en récompense selon la rareté du pet combattu.
Des familiers qui ont des points d'énergie et quelques attaques communes pour faire simple au début, (bonne chance pour faire des attaques variées et dépendantes du pet), et que l'XP de guilde gagné varie en fonction de la rareté du pet vaincu, et un cooldown qui fait qu'après chaque combat, l'énergie du pet est à 0 et se régénère progressivement (toutes les 1-2 minutes, il gagne 5% d'énergie et peut à nouveau combattre s'il a plus de 50% de son énergie au moins).

Voilà, libre à vous d'apporter des suggestions aux idées proposées ou même de nouvelles idées on est preneur !

**Comment by voltou91** [comment]
Moi j'aime bien l'idée des pets pour les combat. Chaque pet pour augmenter (un peu) une de tes stats, par exemple une tortue améliorerait ta défense de +20, un dragon ton attaque de +35. (J'ai dit des chiffres au pif pour imager, je pense pas que ce sera aussi grand).

**Reply by Pietagorh** [reply]
Y'a une autre suggestion pour ça tho

**Comment by KirIcare** [comment]
Concernant la vente de pet à un marchant, le boss s'est prononcé ! Il faudrait trouver un format autre que celui du mini event et si la proposition est suffisamment cohérente, la suggestion aurait plus de chance d'aboutir. Merci d'avance pour vos propositions futures !

**Reply by Muzaraigne** [reply]
Pourquoi ne pas faire un nouvel évent dans un lei , de préférence dans une grande ville , ou le marchand tiendrais une animalerie et ou il proposerai de reprendre  un pet pour de l'xp de guilde ou sur une autre issu d'acheter un pet pour un prix a définir en fonction du nombre d'étoiles , et le taux de proposition de vente serai égal a ce qu'on trouve dans la nature ou un petit peu boosté à vous de voir.

**Comment by Feiryn** [comment]
T'en penses quoi @BastLast ? Ça me paraît être une bonne idée

**Reply by BastLast** [reply]
J'en pensd que ya des bonnes idées dans le thread et d'autres que j'aime moins.

Globalement donner plus d'intérêt aux pets ca serait cool j'aimerai bien aussi que ca soit pas que les pets 5 étoiles qui soient intéressants
